4MD/src/main/java/com/infoshareacademy/four_md/models/Planner.java

```diff
@@ -0,0 +1,34 @@
+package com.infoshareacademy.four_md.models;
```

Comment:
Przenieś tą klasę do model/Planner.java ( jak wykonasz "git merge main" na swojego brancha, to zobaczysz, o czym mówię)